services/jaeger/2.46.0/defaults/cm.yaml

```diff
@@ -14,6 +14,7 @@ data:
           options:
             query:
               base-path: /dkp/jaeger
+          priorityClassName: dkp-high-priority
```

Comment:
I think there's a top-level field we need to set too https://github.com/jaegertracing/helm-charts/blob/7adf270acde7b924aa058cf6ebd46ff5358869c1/charts/jaeger-operator/values.yaml#L96C1-L96C9